core/Types.h

```diff
@@ -786,6 +786,7 @@ class TypeAndOrigins final {
 
     ~TypeAndOrigins() noexcept;
     TypeAndOrigins() = default;
+    TypeAndOrigins(TypePtr type, InlinedVector<Loc, 1> origins) : type(type), origins(origins) {}
```

Comment:
There were a bunch of these where clang required these in places where it didn't before, so i'm not sure when this is required and when it's not to get something like this working:

```
auto foo = TypeAndOrigins{type, origins};
```